apps/extension/babel.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": [],
+  "plugins": ["@babel/plugin-transform-modules-commonjs"]
```

Comment:
Added this one because of the javascript files inside shared rust lib. When we test we set wasm target to node and those js files can't be imported correctly unless we transform them to CommonJs